contracts/src/spaces/entitlements/ICrossChainEntitlement.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// interfaces
+
+// libraries
+
+// contracts
+interface ICrossChainEntitlement {
+  struct Property {
+    string name;
+    string primitive;
+    string description;
+  }
+
+  function isEntitled(
+    address[] calldata users,
+    bytes calldata data
+  ) external view returns (bool);
+
+  function properties() external view returns (Property[] memory);
```

Comment:
I believe parameters is more appropriate, as this is describing parameters that must be passed, encoded in this way, as parameters to the isEntitled call, and not actual properties of the ICrossChainEntitlement as the name properties implies.

Reply:
sounds good, short term memory thought you said properties and not params during our cal. Will fix soon!